docs/api_intro.rst

```diff
@@ -0,0 +1,44 @@
+############
+Introduction
+############
+
+LTD Keeper provides a simple RESTFul API for both administering and browsing documentation resources.
+
+Users and Authentication
+========================
+
+LTD Keeper requires authentication for POST/PUT/PATCH/DELETE operations, but GET requests can be anonymous.
+
+Authenticated API users have a username and password, but a token must be generated for requests.
+See the :doc:`Authentication <auth>` page for information about obtaining a token.
+
+Note that there is no public API for registering new users.
+
+Versioning
+==========
+
+The LTD Keeper API is in beta and can be changed without notice.
+In the future, we may elect to version the API through the ``Accept`` header.
+
```

Comment:
I wouldn't commit to any specifics related to future versioning. Just call it v1 (or the initial version) and leave it at that.

I guess SQRE may really want to have an opinion on how to version future APIs, if that's the case it should be discussed.